### ISSUE TITLE
Get-View TestHelper definition missing parametersets

### ISSUE THREAD

**Issue by lucdekens** [vmware/dscr-for-vmware#75]
The current definition of the Get-View cmdlet in the TestHelper module VMware.VimAutomation.Core, is as follows:

function Get-View {
    param(
        [PSObject] $Server,
        [PSObject] $VIObject
    )

    return $null
}

This doesn't allow using the same definition for other formats of the Get-View cmdlet.
I propose to implement the actual parametersets, present on the actual Get-View cmdlet.

For example (note that this not complete)

function Get-View {
    param(
        [PSObject] $Server,
        [Parameter(ParameterSetName = 'GetViewByVIObject')]
        [PSObject] $VIObject,
        [Parameter(ParameterSetName = 'GetView')]
        [VMware.Vim.ManagedObjectReference] $Id,
        [Parameter(ParameterSetName = 'GetView')]
        [string[]] $Property
    )

    return $null
}

The same remark goes for other PowerCLI cmdlets present in the TestHelpder module(s)

**Comment by SimeonGerginov** [comment]
Yes, for the Get-View cmdlet it will be good to have all parameters implemented. So for now start with the Get-View and then we will discuss other cmdlets which will need all parametersets. I am assigning you to the issue and when you are ready with the changes create new PR.

**Comment by lucdekens** [comment]
I did test that change, but several other resources fail in their Unit tests.
It will require changes on several resources.
Shall I include them in the PR?
